src/main/java/org/carlmontrobotics/robotcode2023/commands/DriveToPoint.java

```diff
@@ -31,16 +32,15 @@ public DriveToPoint(Pose2d targetPose, Drivetrain drivetrain) {
                             drivetrain.getSpeeds()
                         ),
                         PathPoint.fromCurrentHolonomicState(
-                            targetPose,
-                            new ChassisSpeeds(0, 0, 0)
+                            targetPose.get(),
+                            new ChassisSpeeds()
                         )
                     ),
                     drivetrain,
                     new HashMap<>()
                 ).getPathCommand(true, true)
             ),
-            new CorrectToPoint(targetPose, drivetrain)
-
+            new CorrectToPoint(targetPose.get(), drivetrain)
```

Comment:
This will use the targetPose's value at the time DriveToPoint is constructed, not when it is run. Is that what you want?